examples/controlplane/controlplane.yaml

```diff
@@ -107,6 +212,99 @@ spec:
       name: '{{ ds.meta_data["local_hostname"] }}'
       kubeletExtraArgs:
         cloud-provider: azure
+        cloud-config: /etc/kubernetes/azure.json
+  preKubeadmCommands:
+  - bash -c /tmp/kubeadm-bootstrap.sh
+  files:
+  - path: /tmp/kubeadm-bootstrap.sh
```

Comment:
Is this temporary until we have the images built?

Reply:
@vincepri -- yep, definitely!